test/unit/app/controllers/transactions/tx-state-manager-test.js

```diff
@@ -146,6 +146,23 @@ describe('TransactionStateManager', function () {
       assert.equal(result[0].id, 1, 'early txs truncted')
     })
 
+    it('should remove down to the txHistoryLimit if their are more txs then limit in history', function () {
```

Comment:
'should trim the txHistory to the txHistoryLimit' or 
'should trim to the txHistoryLimit if there are more txs than limit in history'

Reply:
```suggestion
    it('should trim to the txHistoryLimit if there are more txs than limit in history', function () {
```